wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
 
     def get(self, url):
+        client_request = self.request.path[1:]
         try:
-            if "/" in url:
-                self.redirect("/index")
-            self.render("index.html", **template.get_context())
+            if self.request.path == "/":
+                if os.path.exists(template.get_path() + "index.html"):
+                    self.render("index.html", **template.get_context())
+                elif os.path.exists(template.get_path() + "index.htm"):
+                    self.render("index.htm", **template.get_context())
+            else:
```

Comment:
@sophron, I understand that a missing index.html file is a reason to stop the attack and warn the attacker, however, I've tried to use sys.exit() and self.stop(), neither of these functions work, am I missing a variable?